config.lua

```diff
@@ -56,6 +56,14 @@ local function simple_incr(a)
 end
```

Comment:
Please rebase to master branch and add changelog entry.

Reply:
Thanks, done.